src/apis/platformvm/api.ts

```diff
@@ -649,6 +650,18 @@ export class PlatformVMAPI extends JRPCAPI {
     return this.callMethod('platform.issueTx', params).then((response:RequestResponseData) => response.data.result.txID);
   };
 
+  getHeight = async ():Promise<BN> => {
+    const params:any = {};
+    return this.callMethod('platform.getHeight', params)
+      .then((response:RequestResponseData) => new BN(response.data.result.minStake, 10));
+  }
+
+  getMinStake = async ():Promise<BN> => {
```

Comment:
`getMinStake` is currently on the `InfoAPI` though Dan said they're going to add it to PlatformVM too.

Also, be aware that there is `platform.getStake` which returns `staked` for a set of addresses.

Reply:
wait so do I put it back on platform or no?

Reply:
getMinStake was moved from Info to Platform because minimum stake amount is ultimately a platform runtime piece of information.